common/app/views/support/cleaner/AmpEmbedCleaner.scala

```diff
@@ -63,7 +63,7 @@ case class AmpEmbedCleaner(article: Article) extends HtmlCleaner {
     def getAmpExternalVideoByUrl(videoUrl: String): Option[AmpExternalVideo] = {
       val youtubePattern = """^https?:\/\/www\.youtube\.com\/watch\?v=([^#&?]+).*""".r
       val vimeoPattern = """^https?:\/\/vimeo\.com\/(\d+).*""".r
-      val facebookPattern = """^https?:\/\/www\.facebook\.com\/(\w+)\/videos\/(\d+)\/""".r
+      val facebookPattern = """^https?:\/\/www\.facebook\.com\/([\w.-]+)\/videos\/(\d+)\/""".r
```

Comment:
I don't think `-` is allowed but `.` is. This is according to the guidelines you linked to.